docs/install_apt.md

```diff
@@ -14,7 +14,7 @@ The NVIDIA package tends to follow more recent library and driver versions, but
 If installing from packages, install the library and latest driver separately; the driver bundled with the library is usually out-of-date.
 This can be skipped for CPU-only installation.
 
-**BLAS**: install ATLAS by `sudo apt-get install libatlas-base-dev` or install OpenBLAS or MKL for better CPU performance.
+**BLAS**: install ATLAS by `sudo apt-get install libatlas-base-dev` or install OpenBLAS (with `sudo apt-get install libopenblas-base libopenblas-dev`) or MKL for better CPU performance.
```

Comment:
No need to include both packages. Just install the `-dev` one and the other gets installed automatically.
```
$ apt-cache depends libopenblas-dev
libopenblas-dev
  Depends: libopenblas-base
  Depends: libblas-dev
```